internal/cli/up.go

```diff
@@ -53,6 +55,14 @@ func (c *upCmd) run(ctx context.Context, args []string) error {
 
 	logOutput(fmt.Sprintf("Starting Tilt (built %s)…\n", buildDateStamp()))
 
+	if trace {
+		traceID, err := tracer.TraceID(ctx)
+		if err != nil {
+			return err
```

Comment:
is this really cause for us to bail out of the whole program?

Reply:
My logic was: If the user is running with --trace, they (presumably? I guess someone might just run with --trace always) want a trace. If we log the error and continue running, it'd be easy to miss the error and then wonder why the trace isn't showing up in honeycomb. If we bail, the error is impossible to miss and if they didn't actually want the trace that badly, they can rerun without --trace.

I don't feel particularly strongly, though.

Reply:
aaaah right, i forgot that tracing was opt-in. this makes sense, leave it as-is!